### PR TITLE
20759948_1 : added Call to Action

### PR DESCRIPTION
**Pull request by amindu00**

### CODE REVIEW

File: src/components/container/CallToAction/CallToAction.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+function CallToAction() {
+    const [selectRole, setSelectRole] = useState(true);
+
+    return (
+        <div className='CallToAction'>
+            <h2>Let's Get Started!</h2>
+            <div className='underLine' />
+            <div className='toggleSwitch__wrapper'>
+                <p>Select your role:</p>
+                <button onClick={() => setSelectRole(!selectRole)} className={'toggleEmplyer ' + (selectRole ? 'toggled' : "")}> <img src={buildingIcon} alt="buildingIcon" /> Employer</button>
```
Comment (jasper-prgh) [inline src/components/container/CallToAction/CallToAction.js]: formatting - lines too long

File: src/components/container/CallToAction/CallToAction.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+                <button onClick={() => setSelectRole(!selectRole)} className={'toggleJob ' + (selectRole ? '' : 'toggled')}> <img src={buildingIcon} alt="buildingIcon" />Job Seeker</button>
+            </div>
+            {selectRole ?
+                <div> Employer stuff here</div> :
+                <div> Job Seeker stuff here</div>}
```
Comment (jasper-prgh) [inline src/components/container/CallToAction/CallToAction.js]: formatting:

```
{selectRole
                ? <div> Employer stuff here</div>
                : <div> Job Seeker stuff here</div>}
```

File: src/components/container/CallToAction/CallToAction.scss [hunk 3]
```diff
@@ -0,0 +1,61 @@
+$Blue_Ribbon: #0066FF;
```
Comment (jasper-prgh) [inline src/components/container/CallToAction/CallToAction.scss]: remove variable and use global css props

File: src/components/container/CallToAction/CallToAction.scss [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+        img {
+            margin: 1px 5px;
+        }
+
+
+        .toggled {
+            background-color: rgba(0, 102, 255, 0.192);
+            border: 1px solid var(--color-blue-ribbon);
+        }
+
```
Comment (jasper-prgh) [inline src/components/container/CallToAction/CallToAction.scss]: remove empty line

File: src/components/container/CallToAction/CallToAction.scss [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+        .toggleJob {
+            border-radius: 0 25px 25px 0;
+        }
+
+        img {
+            margin: 1px 5px;
+        }
+
+
```
Comment (jasper-prgh) [inline src/components/container/CallToAction/CallToAction.scss]: only one empty line

**Review by jasper-prgh — CHANGES_REQUESTED** [review]

File: src/components/container/CallToAction/CallToAction.js [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+function CallToAction() {
+    const [selectRole, setSelectRole] = useState(true);
+
+    return (
+        <div className='CallToAction'>
+            <h2>Let's Get Started!</h2>
+            <div className='underLine' />
+            <div className='toggleSwitch__wrapper'>
+                <p>Select your role:</p>
+                <button onClick={() => setSelectRole(!selectRole)} className={'toggleEmplyer ' + (selectRole ? 'toggled' : "")}>
```
Comment (jasper-prgh) [inline src/components/container/CallToAction/CallToAction.js]: use single-quotes `'something'` everywhere please

**Review by jasper-prgh — CHANGES_REQUESTED** [review]

File: src/components/container/CallToAction/CallToAction.js [hunk 7]
```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
```
Comment (jasper-prgh) [inline src/components/container/CallToAction/CallToAction.js:1]: Really good job though!!

File: src/components/container/CallToAction/CallToAction.js [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+                    Job Seeker
+                </button>
+            </div>
+            {selectRole
+                ? <div> Employer stuff here</div>
+                : <div> Job Seeker stuff here</div>}
+        </div>
+    );
+}
+
```
Comment (tinkabel85) [inline src/components/container/CallToAction/CallToAction.js:30]: don't we want to use here as well BEM classes for the sake of consistency ?

Comment (amindu00) [reply src/components/container/CallToAction/CallToAction.js:30]: didn't get you

Comment (jasper-prgh) [reply src/components/container/CallToAction/CallToAction.js:30]: CallToAction__toggleSwitch-wrapper

**Review by jasper-prgh — CHANGES_REQUESTED** [review]

File: src/components/container/CallToAction/CallToAction.scss [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+        border-radius: 32px;
+        background-color: var(--color-blue-ribbon);
+    }
+
+    .CallToAction__toggleSwitch-wrapper {
+        display: flex;
+        align-items: center;
+
+        p {
+            margin-right: 10px;
```
Comment (tinkabel85) [inline src/components/container/CallToAction/CallToAction.scss:24]: I think we also have to add here the  properties for the font, like:
 color: var(--color-text);
  font-weight: 600;
  font-size: 20px;
  line-height: 2em;
  
  as per provided design

File: src/components/container/CallToAction/CallToAction.scss [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+        }
+
+        button {
+            float: left;
+            background-color: var(--color-white);
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            border: 1px solid #7D7CAC;
```
Comment (tinkabel85) [inline src/components/container/CallToAction/CallToAction.scss]: #7D7CAC is already a global css variable 
  --color-text: #7d7cac;
  
  maybe here as well font properties as per design

File: src/components/container/CallToAction/CallToAction.scss [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            border: 1px solid var(--color-text);
+            padding: 3px 15px;
+
+        }
+
+        .toggleOne {
+            border-radius: 10px 0 0 10px;
```
Comment (tinkabel85) [inline src/components/container/CallToAction/CallToAction.scss]: I think it is border-radius: 16px 0 0 16px; as per design, but it is a minor issue. You can ignore for now

File: src/components/container/CallToAction/CallToAction.scss [hunk 12]
```diff
@@ -0,0 +1,60 @@
...
+        img {
+            margin: 1px 5px;
+        }
+
+        .toggled {
+            background-color: rgba(0, 102, 255, 0.192);
+            border: 1px solid var(--color-blue-ribbon);
+        }
+    }
+}
```
Comment (tinkabel85) [inline src/components/container/CallToAction/CallToAction.scss]: do we want to add media queries for mobile? (the only change here I believe would be to give  p element property display: none;)

**Review by tinkabel85 — APPROVED** [review]

**Review by jasper-prgh — APPROVED** [review]

File: src/components/container/CallToAction/CallToAction.scss (hunk 12 above)
Comment (amindu00) [reply src/components/container/CallToAction/CallToAction.scss]: thanks I didn't see that

**Review by timmy38233 — APPROVED** [review]
Nice work :)

**Review by Puxi19 — APPROVED** [review]

**Pull request merged**